packages/ui-components/stories/UIPersona.story.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import type { IStackTokens } from '@fluentui/react';
+import { Stack, MessageBarType } from '@fluentui/react';
+import { UIPersona, UIPersonaSize } from '../src/components/UIPersona';
+
+export default { title: 'Utilities/Persona' };
+const stackTokens: IStackTokens = { childrenGap: 40 };
+
+export const defaultUsage = (): JSX.Element => {
+    return (
+        <Stack tokens={stackTokens}>
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size24}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size32}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size40}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size48}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size56}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size72}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size100}
+            />
+            <UIPersona
+                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
+                size={UIPersonaSize.size120}
+            />
```

Comment:
```suggestion
            <UIPersona
                imageUrl={`https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-male.png`}
                size={UIPersonaSize.size120}
                text={'John Doe'}
                presence={PersonaPresence.online}
                secondaryText={'Software Engineer'}
                tertiaryText={'In a meeting'}
                optionalText={'Available at 4:00pm'}
            />
```
Could be worth showing other options and maybe export from `PersonaPresence` from UI Components

Reply:
@devinea Done. 
![CleanShot 2022-10-27 at 18 15 51](https://user-images.githubusercontent.com/526128/198356136-d76c33a2-a912-46a9-a3b2-72f73024a016.jpg)